_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After everyone's stomach was filled, the developers were ready to get started in one of the 13 pre-assigned teams, which all addressed different topic areas.
+The whole day was determined by concentration, creativity and of course lots of fun and laughter.
+Some teams got so hooked up and passionate about their project that they even worked until the wee hours of the morning and continued early in the morning the next day.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/HACKATHON?src=hash">#HACKATHON</a> <a href="https://twitter.com/epagesdevs">@epagesdevs</a> at 3.30 AM: Everyone left for beer.. BUT <a href="https://twitter.com/hashtag/smartshop?src=hash">#smartshop</a> hackers are <a href="https://twitter.com/hashtag/unstoppable?src=hash">#unstoppable</a> <a href="https://twitter.com/hashtag/ephack17?src=hash">#ephack17</a> <a href="https://twitter.com/oozpy">@oozpy</a> <a href="https://t.co/xui2qukGNE">pic.twitter.com/xui2qukGNE</a></p>&mdash; Benjamin Nothdurft (@dataduke) <a href="https://twitter.com/dataduke/status/844816040454307840">March 23, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+No wonder then, that the results the next day in the evening were brilliant:
+
+### Final presentations
+
+During the two days of the hackathon the teams mainly focused on pushing forward our new architecture Beyond, enhancing our current shop software ePages 6 and improving internal processes.
+The audience was quite impressed by the creative outcomes of every single team.
+
+For further developing our newest backend architecture some teams dedicated the two days on simplifying the interaction with the Beyond API with the help of the query language [GraphQl](http://graphql.org/learn/).
+Another team made it possible to enable merchants to interact with Beyond by only using natural spoken language by programming Amazon's voice service device [Alexa](https://developer.amazon.com/alexa).
+Yet another team created a dashboard app for visualizing KPIs for the merchant.
+
+One of the teams that was concerned about enhancing our current shop software developed a Amazon mass uploader to upload a bulk of products at once instead of merely one by one.
+
+Internal processes that were improved were, among others, the internal tool we use for our developer blog and REST API, as well as an automatization of the Linux setup for new colleagues starting at ePages.
+
+In the end everyone could vote for his or her favorite result.
+The first three winners were the lucky ones to get Amazon vouchers.
+
+The team that made the first place, was the one that developed a new stylish theme for our current core shop software ePages 6 as well as Beyond.
```

Comment:
The winner team's project was the development of a new stylish theme. New features that came with this team were, e.g. modern ..., and much more.